validator/validators/feature/feature_validator.go

```diff
@@ -31,25 +38,59 @@ func (s *FeatureValidator) GenerateLoad() error {
 		dataRate              = s.vConfig.GetDataRate()
 		agentCollectionPeriod = s.vConfig.GetAgentCollectionPeriod()
 		agentConfigFilePath   = s.vConfig.GetCloudWatchAgentConfigPath()
-		receiver              = s.vConfig.GetPluginsConfig()
+		receivers             = s.vConfig.GetPluginsConfig()
 	)
 
-	err := common.StartLogWrite(agentConfigFilePath, agentCollectionPeriod, dataRate)
-	if err != nil {
+	if err := common.StartLogWrite(agentConfigFilePath, agentCollectionPeriod, dataRate); err != nil {
 		multiErr = multierr.Append(multiErr, err)
 	}
 
 	// Sending metrics based on the receivers; however, for scraping plugin  (e.g prometheus), we would need to scrape it instead of sending
-	err = common.StartSendingMetrics(receiver, agentCollectionPeriod, dataRate)
-	if err != nil {
-		multiErr = multierr.Append(multiErr, err)
+	for _, receiver := range receivers {
```

Comment:
Feature validation is cover all of the use case for a given OS. Therefore, we don't want to use a single receiver here (e.g prometheus, emf) unlike [all other validator](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/validators/performance/performance_validator.go#L53-L59)